src/main/java/main.java

```diff
@@ -0,0 +1,7 @@
+import duke.Duke;
+
+public class main {
```

Comment:
```suggestion
public class Main {
```
I really like that you extracted the entry point of the application into its own class. It is very clean. Hmm but perhaps the class name main could be capitalised to follow the coding standard?